uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/feature/federatedlogin/testscripts/FederatedLogin.java

```diff
@@ -0,0 +1,350 @@
+import org.springframework.http.*;
+import org.springframework.security.oauth2.client.test.TestAccounts;
+import org.springframework.security.oauth2.common.util.RandomValueStringGenerator;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestOperations;
+import org.springframework.web.client.RestTemplate;
+
+import java.net.Inet4Address;
+import java.net.UnknownHostException;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.cloudfoundry.identity.uaa.provider.saml.SamlKeyManagerFactoryTests.*;
+import static org.cloudfoundry.identity.uaa.provider.saml.SamlKeyManagerFactoryTests.certificate2;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.*;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = DefaultIntegrationTestConfig.class)
+public class FederatedLogin {
+
+    public static final String IDP_ENTITY_ID = "cloudfoundry-saml-login";
+
+    private final SamlTestUtils samlTestUtils = new SamlTestUtils();
+    @Autowired
+    @Rule
+    public IntegrationTestRule integrationTestRule;
+
+    @Autowired
+    RestOperations restOperations;
+
+    @Autowired
+    WebDriver webDriver;
+
+    @Value("${integration.test.base_url}")
+    String baseUrl;
+
+    @Autowired
+    TestAccounts testAccounts;
+
+    @Autowired
+    TestClient testClient;
+
+    ServerRunning serverRunning = ServerRunning.isRunning();
+
+    //*********Loading all data from constants file**********//
+    static String ExpectedUrl = Constants.URL;
+    static String GESSO_UserName = Constants.SS0_UserName;
+    static String GESSO_Password = Constants.SS0_Password;
+    static String IDP_Password = Constants.IDP_Password;
+    static String IDP_UserName = Constants.IDP_UserName;
+
+    @Before
+    public void clearWebDriverOfCookies() throws Exception {
+        samlTestUtils.initialize();
+        webDriver.get(baseUrl + "/logout.do");
+        webDriver.manage().deleteAllCookies();
+        webDriver.get(baseUrl.replace("localhost", "testzone1.localhost") + "/logout.do");
+        webDriver.manage().deleteAllCookies();
+        webDriver.get(baseUrl.replace("localhost", "testzone2.localhost") + "/logout.do");
+        webDriver.manage().deleteAllCookies();
+        assertTrue("Expected testzone1.localhost and testzone2.localhost to resolve to 127.0.0.1", doesSupportZoneDNS());
+    }
+
+    @Before
+    public void setup() {
+        String token = IntegrationUtilsOrchestrator.getClientCredentialsToken(baseUrl, "admin", "adminsecret");
+
+        ScimGroup group = new ScimGroup(null, "zones.testzone1.admin", null);
+        IntegrationUtilsOrchestrator.createGroup(token, "", baseUrl, group);
```

Comment:
That looks like IntegrationUtilsOrchestrator.createGroup() is a duplicate of [IntegrationTestUtils.createGroup()](https://github.com/GESoftware-CF/uaa/blob/orchestrator_intergration_test/uaa/src/test/java/org/cloudfoundry/identity/uaa/integration/util/IntegrationTestUtils.java#LL549C1-L569C6l).  Use the one from IntegrationTestUtils

Reply:
Done